Tests/DatadogTests/Datadog/RUM/RUMVitals/VitalRefreshRateReaderTests.swift

```diff
@@ -0,0 +1,104 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import XCTest
+import UIKit
+@testable import Datadog
+
+class VitalRefreshRateReaderTests: XCTestCase {
+    private let mockNotificationCenter = NotificationCenter()
+
+    func testRefreshRateReader() throws {
+        let reader = VitalRefreshRateReader(notificationCenter: mockNotificationCenter)
+        XCTAssertFalse(reader.isRunning)
```

Comment:
The `isRunning` property seems very internal to the component we test. IMO it should be made `private` and instead, we should be testing the properties / behaviours of "refresh rate monitor":
* when UI is clunky, then measured mean FPS is lower than `UIScreen.main.maximumFramesPerSecond`;
* when UI is smooth, then measured mean FPS is about `UIScreen.main.maximumFramesPerSecond`;
* when app goes to background and then back to foreground, then measured mean FPS is not impacted.